env specific data custom env (#331)
getgauge/docs.gauge.org: configuration.rst

.. attention::
   You can either create a new ``.properties`` file or use an existing file to add your custom key value pairs.

Custom location for Environment Property files
----------------------------------------------

.. versionadded:: 1.4.0

> Gauge allows you to choose a location within the project directory where the environment files are located. By default, Gauge looks for these files in ``env`` directory.

Reviewer: ```suggestion
By default, Gauge looks for environment property files in the ``env`` directory of the project. 
```

The custom location can be using the below options

* Set the location of the directory containing the environment property files in ``gauge_env_dir`` environment variable.
* Set ``EnvironmentDir`` key in ``manifest.json`` to the location of the directory containing the environment property files.

> When both of the above are missing, or if ``gauge_env_dir`` is unset and ``manifest.json`` is missing, gauge will default to ``env``.

Reviewer: ```suggestion
If ``gauge_env_dir`` is unset and ``EnvironmentDir`` is missing in ``manifest.json``, gauge will default to ``env``.
```
